server.js

```diff
@@ -2719,6 +2719,84 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Cocoapods Downloads integration.
+camp.route(/^\/cocoapods\/(dm|dw|dt)\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var info = match[1]; // One of these: "dm", "dw", "dt"
+  var spec = match[2];  // eg, AFNetworking
+  var format = match[3];
+  var apiUrl = 'http://metrics.cocoapods.org/api/v1/pods/' + spec;
+  var badgeData = getBadgeData('pod', data);
```

Comment:
'downloads' instead of 'pod'. It incidentally removes the need for `getLabel()` below.